src/components/SurveySelectField/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import SelectSearch from 'react-select-search/dist/cjs';
+
+const SurveySelectField = ({ options }) => {
```

Comment:
@rafayet-monon Should we naming this to be more generic without prefix with `survey`?
So we can use this component in anywhere ✌️
Naming something like this component `FormSelectField`

Reply:
I kind of plan to use these components only for the survey as there will be some logic and operations in each of these. That's why I prefixed them as survey.